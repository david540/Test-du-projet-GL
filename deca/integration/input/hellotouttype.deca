{
    println(2.0);
    print(2.0);
    println();
    printx(2.0);
    println();
    printlnx(2.0);
}
