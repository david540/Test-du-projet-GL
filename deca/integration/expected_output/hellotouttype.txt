2.00000e+00
2.00000e+00
0x1p+1
0x1p+1
